Gruntfile.js

```diff
@@ -116,6 +116,7 @@ module.exports = function (grunt) {
             'lib/patternfly/components/d3/d3.js',
             'lib/angular/angular.js',
             'lib/angular-animate/angular-animate.js',
+            'lib/lodash/lodash.min.js',
```

Comment:
I don't love the addition of lodash here even though I am supportive of using the library within applications. We need to make sure that future commits don't use functions that are in lodash but not in underscore since these need to be interchangeable.